features/users/user_auth.feature

```diff
@@ -36,7 +36,7 @@ Feature: User Authentication
   Scenario: A user is able to login successfully with provided credentials
     Given I am on the login page
     And I am admin
-    Then I should see "Admin" within "#top-menu-items"
+    Then I should find ".icon-user" within "#top-menu-items"
```

Comment:
This change leads to cukes that are way harder to read. Please write a small step definition, e.g.

```
Then I should see "Admin" as being logged in
```

Let me know if you need pointers as to how it is done.